eng/Versions.props

```diff
@@ -52,4 +52,15 @@
     <HumanizerVersion>2.2.0</HumanizerVersion>
     <XunitCombinatorialVersion>1.2.7</XunitCombinatorialVersion>
   </PropertyGroup>
+
+  <PropertyGroup>
+    <!--
+      Binaries that need to be executable during source build are restricted to versions available within source build.
+      This section defines executable versions of packages which are referenced via lower-version reference assemblies
+      during the build.
+    -->
+    <SystemCollectionsImmutableExecutableVersion>5.0.0</SystemCollectionsImmutableExecutableVersion>
+    <SystemReflectionMetadataExecutableVersion>5.0.0</SystemReflectionMetadataExecutableVersion>
+    <MicrosoftCodeAnalysisExecutableVersion>3.8.0</MicrosoftCodeAnalysisExecutableVersion>
```

Comment:
Is there a way we can have arcade auto-update these dependencies so we don't need to be responsible for updating them? Logically these _should_ be whatever compiler is being produced as part of source build right?

Reply:
Arcade should not be responsible for this. If this is a necessary concept for tooling, it should be pushed all the way up to dotnet CLI and/or nuget to allow execution of a project using a different (newer) set of dependencies than the build used.